### PR TITLE
Update to hunts global

### PR DESCRIPTION
**Pull request by cocosolos**
Move hunt text into their respective zones ID file. Change table structure for hunts global.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: documentation/Regime_IDs.csv [hunk 1]
```diff
@@ -0,0 +1,848 @@
...
+  552,    Hunt Registry,    ,    ,    Farruca Fly
+  553,    Hunt Registry,    ,    ,    Verthandi
+  554,    Hunt Registry,    ,    ,    Urd
+  555,    Hunt Registry,    ,    ,    Skuld
+  556,    Hunt Registry,    ,    ,    Lord Ruthven
+  557,    Hunt Registry,    ,    ,    Erebus
+  558,    Hunt Registry,    ,    ,    Feuerunke
+  559,    Hunt Registry,    ,    ,    Yilbegan
+"RegimeID", "Regime Type", "Page", "Book Area(s)/NM Area(s)", "Target(s)"
+  660,    Dominion OPs,    ,    ,
```
Comment (Xaver-DaRed) [inline documentation/Regime_IDs.csv]: 560 instead of 660

Comment (cocosolos) [reply documentation/Regime_IDs.csv]: Good eye, thanks!

**Comment by kaincenteno** [comment]
will this cause an issue with the June ID Shift that hasnt been merged yet?

**Comment by cocosolos** [comment]
> will this cause an issue with the June ID Shift that hasnt been merged yet?

That is a good point. It's possible the text IDs were shifted.

**Review by ibm2431 — APPROVED** [review]

**Comment by ibm2431** [comment]
Judging by the shifts in the June version update, I don't think these would be affected~!

**Pull request merged**